util/storage/storage.go

```diff
@@ -1,121 +1,60 @@
 package storage
 
 import (
-	"strings"
-	"time"
-
-	"github.com/orourkedd/influxdb1-client/client"
-	"github.com/src-d/envconfig"
 	"golang.org/x/tools/benchmark/parse"
 	"gopkg.in/src-d/go-errors.v1"
-	"gopkg.in/src-d/go-log.v1"
 )
 
-const envPrefix = "influx"
+const (
+	// PerOpSeconds represents metric of seconds per operation
+	PerOpSeconds = "per_op_seconds"
+	// PerOpAllocBytes represents metric of bytes allocated per operation
+	PerOpAllocBytes = "per_op_alloc_bytes"
+	// PerOpAllocs represents metric of allocations per operation
+	PerOpAllocs = "per_op_allocs"
+)
 
-// InfluxClient embeds influxdb client itself and also contains the configuration info
-type InfluxClient struct {
-	client.Client
-	influxConfig influxConfig
-}
-
-type influxConfig struct {
-	Address  string
-	Username string
-	Password string
-	Db       string
-	// Measurement acts as a container for tags, fields, and the time column, and the measurement name is the description of the data that are stored in the associated fields.
-	// Measurement names are strings, and, for any SQL users out there, a measurement is conceptually similar to a table.
-	Measurement string
-}
+// Constructor is a type that represents function of default storage client constructor
+type Constructor func() (Client, error)
 
 var (
-	errGetClientFailed = errors.NewKind("cannot get influx db client")
-	errDumpFailed      = errors.NewKind("cannot dump batch points")
-)
+	// Constructors is a map of all supported storage client constructors
+	Constructors = make(map[string]Constructor)
```

Comment:
If it is used only inside the single package, I wonder if it has to be exported here?

Reply:
Done